package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.15",
     "@docusaurus/preset-classic": "^2.0.0-beta.15",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.15",
+    "@docusaurus/theme-search-algolia": "^2.0.0-beta.17",
```

Comment:
this change seems unrelated to this PR, can we skip this?

Reply:
Ok